src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { TabProps } from '../../utils/types';
+
+export const Tab: React.FC<TabProps> = ({
+  isActive,
+  title,
+  id,
+  setActiveTabId,
+}) => {
+  return (
+    <li className={isActive ? 'is-active' : ''} data-cy="Tab">
+      <a
+        href={`#${id}`}
+        data-cy="TabLink"
+        onClick={() => setActiveTabId(id)}
```

Comment:
You are calling setActiveTabId directly in the onClick. It is recommended to make a separate function handleTabClick and call setActiveTabId from that function.